### DISCUSSION TITLE
Disable page cache on template

### DISCUSSION THREAD

**Discussion by robsonsobral** [ExpressionEngine/ExpressionEngine#2966] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
It's more like... I've never thought of it, but could be awesome if we could control page cache from the template itself. I'm thinking on that templates that should **never** be cached, but they could by mistake.

**Describe the solution you'd like**
In case the first line of a template has a specific markup, the cache would be bypassed

**Teachability, Documentation, Adoption, Migration Strategy**

```html
{!-- DO NOT CACHE --}
```

- [x] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
FWIW, ExpressionEngine has (undocumented) support of non-cached comment form within cached template.
`'{NOCACHE_COMMENT_FORM="' . $nc . '"}' . ee()->TMPL->tagdata . '{/NOCACHE_FORM}'`
It might be worth to expand that to other forms and / or specific markup as suggested above
